test/evaluate/ISO-Fortran-binding.cc

```diff
@@ -79,8 +80,9 @@ static void AddNoiseToCdesc(CFI_cdesc_t *dv, CFI_rank_t rank) {
 }
 
 #ifdef VERBOSE
-static void DumpTestWorld(void *bAddr, CFI_attribute_t attr, CFI_type_t ty,
-    std::size_t eLen, CFI_rank_t rank, CFI_index_t *eAddr) {
+static void DumpTestWorld(const void *bAddr, CFI_attribute_t attr,
```

Comment:
A void * object cannot be dereferenced, so it's not clear that adding "const" really adds much.